src/lib/snark_worker/standalone/run_snark_worker.ml

```diff
@@ -8,6 +8,7 @@ let command =
     (let%map_open spec =
        flag "--spec-sexp" ~doc:""
          (required (sexp_conv Prod.single_spec_of_sexp))
+     and network_constants = Cli_lib.Flag.network_constants
```

Comment:
TODO: ensure the code that calls worker from the Mina node is updated. Also making `--network` obligatory for the snark workers may actually be troublesome for snark worker operators, we'll need to make not on that when releasing